src/app/screens/SignUp/components/CustomInput/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import styles from './styles.module.scss';
+
+interface Props extends React.HTMLProps<HTMLInputElement> {
+  inputRef?: (ref: HTMLInputElement) => void;
+  error?: string;
+}
+
+export default function CustomInput({ inputRef, error, ...props }: Props) {
+  return (
+    <>
```

Comment:
you ssure this doesn't need a div container with `column` global class? also with the full-width and another standard for the app?